kolibri/plugins/learn/assets/src/views/ContentPage.vue

```diff
@@ -328,6 +327,20 @@
           params: { id },
         };
       },
+      launchIntent() {
+        return client.post({
+          path: urls['kolibri:kolibri.plugins.app:appcommands-launch_intent'](),
```

Comment:
I feel like maybe these URLs should somehow "go through" (be generated by) something more centralized in the "capabilities" system, rather than being somewhat "hard-coded" here. Just trying to think about how to keep the cross-plugin interfaces as clean/robust as possible. Just flagging in case you might have ideas for that.

Reply:
Yeah, can make a helper function for this.